samples/nextjs/sitecore/config/JssNextWeb.config

```diff
@@ -79,13 +79,10 @@
 
           There are many available attributes, and they inherit the defaults if not explicitly specified here.
           Defaults are defined in `/App_Config/Sitecore/JavaScriptServices/Sitecore.JavaScriptServices.Apps.config`
-
-          NOTE: graphQLEndpoint enables _Integrated GraphQL_. If not using integrated GraphQL, it can be removed.
```

Comment:
@anastasiya29 Integrated GraphQL is not related to Integrated Mode. So, In connected mode your Integrated Graphql component is still working

Reply:
Yeah, this all needs to stay here for integrated GraphQL to function properly